gcloud/datastore/connection.py

```diff
@@ -421,7 +421,7 @@ def allocate_ids(self, dataset_id, key_pbs):
         return list(response.key)
 
     def save_entity(self, dataset_id, key_pb, properties,
```

Comment:
This was in reference to `save_entity`. I am in the split view on GitHub. Does this show up elsewhere for you?